rakefile.rb

```diff
@@ -95,6 +101,9 @@
 desc "Execute default tasks"
 task :default => [ :vars, :gitlink, :unit, :integ, :spec, :pack ]
 
+desc "Execute default tasks for .NET Core solution"
+task :dotnet => [ :default, :unit_dotnet, :pack_dotnet]

```

Comment:
I should update the description. Unless you'd rather this target _not_ run the regular stuff.